src/vma/vma_extra.h

```diff
@@ -421,6 +543,37 @@ struct __attribute__ ((packed)) vma_api_t {
 	 * @return 0 on success, or error code on failure.
 	 */
 	int (*dump_fd_stats) (int fd, int log_level);
+
+	/**
+	 * Get data from the MP_RQ cyclic buffer
+	 * @param fd - the fd of the ring to query - get it using @ref get_socket_rings_fds
```

Comment:
Please change: `get it using` --> `obtained by`

#### Introduced in:
`add cyclic function api and enalbe related functions when not using VMA_POLL`